lib/minfraud/assessments.rb

```diff
@@ -129,18 +136,22 @@ def score
     private
 
     def perform_request(endpoint)
-      raw = request.perform(
-        verb:     :post,
-        endpoint: endpoint.to_s,
-        body:     request_body,
-      )
+      response = nil
+      body     = nil
+      Minfraud.connection_pool.with do |client|
+        response = client.post(
+          "/minfraud/v2.0/#{endpoint}",
+          body: JSON.generate(request_body),
+        )
+
+        body = response.to_s
```

Comment:
Potentially, we could use `response.parse(:json)` here, but it looks that would require additional changes.

Reply:
Yeah, probably! The docs aren't great about how to use that but I imagine it would be doable. I'm inclined to leave it though as it is clear that the body gets fully read this way, which is important for reusing connections.